tests/openssh.rs

```diff
@@ -9,6 +9,10 @@ fn addr() -> String {
     std::env::var("TEST_HOST").unwrap_or("ssh://test-user@127.0.0.1:2222".to_string())
 }
 
+fn path() -> String {
```

Comment:
Hmm, I think for the purposes of the test, we want to test a specific path. Something like
```rust
control_directory("control-test")
```
I don't think we need to be able to control it through an environment variable. You may need to create the control dir first, I'm not sure.

Reply:
You do indeed, that's why I left it to an env variable but I can change that approach and create the dir beforehand.